docs/specifications/tests/DNSSEC-TP/dnssec01.md

```diff
@@ -15,6 +15,10 @@ If [RFC 8624] and the [IANA registry] disagree on the same DS digest
 algorithm, the RFC takes precedence until the registry has a been 
 updated with a reference to the RFC.
 
+At the time of writing (2020-04-08), algorithm 1 (SHA-1) is still
```

Comment:
"At the time of writing (2020-04-08)" -> "As of 2020-04-08"?

Reply:
Yes, that is an alternative. Both works. None is better than the other.